packages/api/lambdas/sf-scheduler.js

```diff
@@ -3,64 +3,67 @@
 const get = require('lodash.get');
 const merge = require('lodash.merge');
 const uuidv4 = require('uuid/v4');
-const { getS3Object, parseS3Uri } = require('@cumulus/common/aws');
+const { getMessageFromTemplate } = require('@cumulus/ingest/queue');
 const { SQS } = require('@cumulus/ingest/aws');
 const { Provider, Collection } = require('../models');
 
-/**
- * Builds a cumulus-compatible message and adds it to the startSF queue
- * startSF queue will then start a stepfunction for the given message
- *
- * @param   {Object} event   - lambda input message
- * @param   {Object} context - lambda context
- * @param   {function} cb    - lambda callback
- */
-function schedule(event, context, cb) {
-  const template = get(event, 'template');
+const buildCumulusMeta = (queueName) => ({
+  execution_name: uuidv4(),
+  queueName
+});
+
+async function getProvider(providerId) {
+  if (providerId) {
+    const p = new Provider();
+    return p.get({ id: providerId });
+  }
+  return undefined;
```

Comment:
Wouldn't just `return` be sufficient here?

Reply:
lint doesn't like `return`, but it does like `return undefined`

Reply:
If you do an early return at the beginning does it still complain?

`if (!providerId) return;`